src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -374,1065 +349,576 @@
-        // Effective Date may be in the future (eg, for BCOMP COA filings).
-        // If Effective Date is empty, use Filing Date instead.
-        const effectiveDate = header.effectiveDate
-          ? this.apiToPacificDate(header.effectiveDate)
-          : filingDate
-
-        // is this a BCOMP Future Effective Change of Address?
-        const isBcompCoaFutureEffective = this.isBComp &&
-          (header.status === FilingStatus.PAID) &&
-          (filingType === FilingTypes.CHANGE_OF_ADDRESS) &&
-          this.isEffectiveDateFuture(filing)
-
-        // build filing item
-        const item: HistoryItemIF = {
-          filingType,
-          title: this.filingTypeToName(filingType),
-          filingId: header.filingId,
-          filingAuthor: header.certifiedBy,
-          filingDate: this.isStaffFiling(filingType) ? filingDateTime : filingDate, // special case
-          effectiveDate, // used for BCOMP COA Future Effective tooltip
-          isBcompCoaFutureEffective,
-          isPaid: (header.status === FilingStatus.PAID),
-          documents: filing?.documents || [] as Array<any>,
-          status: header.status,
-          isCorrected: (header.isCorrected || false),
-          isCorrectionPending: (header.isCorrectionPending || false),
-          comments: this.flattenAndSortComments(header.comments)
-        }
-
-        // Apply additional properties for Staff Only Filings
-        if (this.isStaffFiling(filingType)) {
-          let baseFiling
-          switch (filingType) {
-            case FilingTypes.REGISTRARS_NOTATION: baseFiling = filing.registrarsNotation
-              break
-            case FilingTypes.REGISTRARS_ORDER: baseFiling = filing.registrarsOrder
-              break
-            case FilingTypes.COURT_ORDER: baseFiling = filing.courtOrder
-              break
-          }
-
-          item.notationOrOrder = baseFiling?.orderDetails
-          item.fileNumber = baseFiling?.fileNumber
-          item.planOfArrangement = baseFiling?.effectOfOrder ? 'Pursuant to a Plan of Arrangement' : ''
 
-          item.filingAuthor = this.isRoleStaff ? header.submitter : 'Registry Staff'
-        }
-
-        // add receipt
-        if (header.paymentToken) {
-          item.documents.push({
-            type: this.DOCUMENT_TYPE_RECEIPT,
-            corpName: this.entityName || this.getCorpTypeNumberedDescription(this.entityType),
-            filingDateTime,
-            paymentToken: header.paymentToken,
-            title: 'Receipt',
-            filename: `${this.entityIncNo} - Receipt - ${filingDate}.pdf`
-          })
-        }
-
-        this.historyItems.push(item)
-      } else {
-        // eslint-disable-next-line no-console
-        console.log('ERROR - missing section in filing =', filing)
+      // special handling for paper-only filings
+      if (filing.availableOnPaperOnly) {
+        // clear URL so we don't try to load documents
+        item.documentsLink = null
```

Comment:
so the API will return a documents link even for paper filings?

Reply:
I don't know yet :)

This code clears this property so that we don't try to fetch documents for this filing (when the user clicks to expand this item).

I'm torn whether I should leave this (as a safety) or delete it (if the API doesn't provide it). Thoughts?

Reply:
I was just curious.  I think we can keep it just to be safe.